registry/purview-registry/registry/models.py

```diff
@@ -113,9 +113,18 @@ def __str__(self):
 class RelationshipType(Enum):
     Contains = 1
     BelongsTo = 2
-    Consumes = 3
-    Produces = 4
+    Consumes = 4
```

Comment:
Might need @YihuiGuo 's input here on those changes. I'm fine with it.

Reply:
I made this change to support the bit operation ' | '. It can make comparison simpler.

Reply:
LGTM, @enya0405 pls make sure you run the test in `registry/purview-registry/test` , espeically `test_creation` and `test_get`.

Reply:
Tests done